tests/functionality/opflow/selfcheck.cpp

```diff
@@ -24,8 +25,9 @@ struct OpflowFunctionalityTestParameters {
   double power_imbalance_penalty = 1000.0;
   std::string description = "";
   int hiop_verbosity_level = 0;
-  std::string hiop_compute_mode;
-  std::string hiop_mem_space;
+  std::string hiop_compute_mode =
+      OPFLOWOptions::hiop_compute_mode.default_value;
+  std::string hiop_mem_space = OPFLOWOptions::hiop_mem_space.default_value;
```

Comment:
I don't immediately see anything wrong with these changes, but is there a motivation for default initializing these? How did this work beforehand?

Reply:
These values overwrite the default values in the OPFLOW module. If you don't initialize these values to something and they aren't included in the .toml file (which would cause them to be initialized to a specific value), then you are using randomly initialized data to start the calculation off. This will cause problems.